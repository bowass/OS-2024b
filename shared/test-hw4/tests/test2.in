7120
10665
11932
8840
900
65513
50903
49522
48131
21163
56537
5464
4273
14501
50432
54443
13156
7013
7095
174
8816
20016
5533
7109
33417
48521
43235
45761
32118
28377
64865
53366
4640
45367
59757
551
28064
26303
5472
29596
2929
692
9319
47115
50183
44612
49871
2349
35767
64396
11765
56070
40417
49593
38431
63237
24947
56477
25017
43784
46008
32967
24693
23480
38639
59446
46041
6023
48256
26034
26518
10560
25038
56109
57468
39821
5305
36429
43850
43019
49657
45023
44828
37304
24523
30978
46943
10025
62233
27594
36455
31717
7713
4623
4147
9448
49721
1355
8712
44676
64373
46055
4607
27793
18264
62988
24757
36540
63777
6702
37186
20615
63076
54789
15918
7391
12400
17070
17724
30526
61362
6465
19152
11513
9937
14261
10619
61297
31119
1449
49252
47383
10629
23193
55555
5560
27701
37564
8788
53055
23745
44417
61461
3203
59594
62068
30800
53634
4207
50845
22302
293
53369
24449
40355
49537
19095
61236
19457
5602
63850
31394
46940
54093
10163
2132
54261
14317
26976
26031
44302
827
11637
25051
6309
30890
1764
19165
62298
65340
28983
17125
55114
5201
37633
21240
1469
64060
37524
10922
39244
64327
46599
62366
30856
27379
57114
60756
58597
11775
54087
23920
17490
8672
14850
48421
28777
2202
51773
9937
9462
33618
56581
52742
65221
14918
19001
62259
38283
39452
938
12416
47769
1358
47135
2726
54628
65331
1944
17572
18578
57948
41736
29141
55860
51119
15412
61733
52952
60013
52441
8730
62644
36036
61231
54219
46876
26634
1010
56291
27858
7079
16577
41965
25390
649
23687
26327
37770
48850
55269
20960
63889
1722
49700
46511
25058
54231
53399
20496
45816
39825
54606
40731
12662
53026
63915
10997
17026
40648
12876
9512
36323
48874
63581
41748
42116
43507
12003
8888
16545
30047
8536
17013
40235
42330
48778
58849
40072
57650
26395
32920
12775
32550
64332
57011
27674
61976
24581
16144
38757
36062
55791
7718
65402
32832
37241
22318
4281
51604
41222
29255
41321
39423
20362
2544
6528
44613
24497
833
48821
45667
41637
12164
4978
13532
42676
5090
27504
62339
16686
30105
35957
57929
25324
8379
10870
41775
43375
3936
22083
31042
55813
38568
1545
3319
12109
40105
8305
15698
17842
3567
21480
21221
35554
54979
41431
3210
39122
22362
62037
16715
52669
63744
21962
43738
41534
25703
14716
41044
38642
40037
14664
29874
32966
43849
5997
1660
53649
20978
52720
51913
21267
40187
48954
35481
30673
1035
25866
32929
20616
50548
20481
62932
18451
63785
36663
31970
21530
20887
54916
37491
26145
11008
6844
30250
20151
11199
36746
37623
60766
48445
19145
2965
33220
4189
31091
25406
3979
41405
11331
50046
21407
51369
18681
64074
63766
22692
29343
53581
18051
49651
49492
12900
30143
24515
13743
37710
2505
13139
30534
17501
6435
3490
30492
9528
59657
20163
27456
23329
46241
52075
65269
38035
17452
41142
12719
6329
17336
61261
26162
46339
52074
63021
48633
57063
26270
26307
49500
63436
63619
18653
48786
27685
54925
24081
44804
39591
59359
41151
14000
20027
21998
16131
63023
18275
53500
54340
27093
7229
