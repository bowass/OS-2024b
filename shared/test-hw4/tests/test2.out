Virtual address: 7120 Physical address: 208 Value: 0
Virtual address: 10665 Physical address: 425 Value: 0
Virtual address: 11932 Physical address: 668 Value: 0
Virtual address: 8840 Physical address: 904 Value: 0
Virtual address: 900 Physical address: 1156 Value: 0
Virtual address: 65513 Physical address: 1513 Value: 0
Virtual address: 50903 Physical address: 1751 Value: -75
Virtual address: 49522 Physical address: 1906 Value: 48
Virtual address: 48131 Physical address: 2051 Value: 0
Virtual address: 21163 Physical address: 2475 Value: -86
Virtual address: 56537 Physical address: 2777 Value: 0
Virtual address: 5464 Physical address: 2904 Value: 0
Virtual address: 4273 Physical address: 3249 Value: 0
Virtual address: 14501 Physical address: 3493 Value: 0
Virtual address: 50432 Physical address: 3584 Value: 0
Virtual address: 54443 Physical address: 4011 Value: 42
Virtual address: 13156 Physical address: 4196 Value: 0
Virtual address: 7013 Physical address: 101 Value: 0
Virtual address: 7095 Physical address: 183 Value: -19
Virtual address: 174 Physical address: 4526 Value: 0
Virtual address: 8816 Physical address: 880 Value: 0
Virtual address: 20016 Physical address: 4656 Value: 0
Virtual address: 5533 Physical address: 2973 Value: 0
Virtual address: 7109 Physical address: 197 Value: 0
Virtual address: 33417 Physical address: 5001 Value: 0
Virtual address: 48521 Physical address: 5257 Value: 0
Virtual address: 43235 Physical address: 5603 Value: 56
Virtual address: 45761 Physical address: 5825 Value: 0
Virtual address: 32118 Physical address: 6006 Value: 31
Virtual address: 28377 Physical address: 6361 Value: 0
Virtual address: 64865 Physical address: 6497 Value: 0
Virtual address: 53366 Physical address: 6774 Value: 52
Virtual address: 4640 Physical address: 6944 Value: 0
Virtual address: 45367 Physical address: 7223 Value: 77
Virtual address: 59757 Physical address: 7533 Value: 0
Virtual address: 551 Physical address: 7719 Value: -119
Virtual address: 28064 Physical address: 8096 Value: 0
Virtual address: 26303 Physical address: 8383 Value: -81
Virtual address: 5472 Physical address: 2912 Value: 0
Virtual address: 29596 Physical address: 8604 Value: 0
Virtual address: 2929 Physical address: 8817 Value: 0
Virtual address: 692 Physical address: 7860 Value: 0
Virtual address: 9319 Physical address: 9063 Value: 25
Virtual address: 47115 Physical address: 9227 Value: 2
Virtual address: 50183 Physical address: 9479 Value: 1
Virtual address: 44612 Physical address: 9796 Value: 0
Virtual address: 49871 Physical address: 10191 Value: -77
Virtual address: 2349 Physical address: 10285 Value: 0
Virtual address: 35767 Physical address: 10679 Value: -19
Virtual address: 64396 Physical address: 10892 Value: 0
Virtual address: 11765 Physical address: 11253 Value: 0
Virtual address: 56070 Physical address: 11270 Value: 54
Virtual address: 40417 Physical address: 11745 Value: 0
Virtual address: 49593 Physical address: 1977 Value: 0
Virtual address: 38431 Physical address: 11807 Value: -121
Virtual address: 63237 Physical address: 12037 Value: 0
Virtual address: 24947 Physical address: 12403 Value: 92
Virtual address: 56477 Physical address: 2717 Value: 0
Virtual address: 25017 Physical address: 12473 Value: 0
Virtual address: 43784 Physical address: 12552 Value: 0
Virtual address: 46008 Physical address: 12984 Value: 0
Virtual address: 32967 Physical address: 13255 Value: 49
Virtual address: 24693 Physical address: 13429 Value: 0
Virtual address: 23480 Physical address: 13752 Value: 0
Virtual address: 38639 Physical address: 12015 Value: -69
Virtual address: 59446 Physical address: 13878 Value: 58
Virtual address: 46041 Physical address: 13017 Value: 0
Virtual address: 6023 Physical address: 14215 Value: -31
Virtual address: 48256 Physical address: 2176 Value: 0
Virtual address: 26034 Physical address: 14514 Value: 25
Virtual address: 26518 Physical address: 14742 Value: 25
Virtual address: 10560 Physical address: 320 Value: 0
Virtual address: 25038 Physical address: 12494 Value: 24
Virtual address: 56109 Physical address: 11309 Value: 0
Virtual address: 57468 Physical address: 14972 Value: 0
Virtual address: 39821 Physical address: 15245 Value: 0
Virtual address: 5305 Physical address: 15545 Value: 0
Virtual address: 36429 Physical address: 15693 Value: 0
Virtual address: 43850 Physical address: 12618 Value: 42
Virtual address: 43019 Physical address: 5387 Value: 2
Virtual address: 49657 Physical address: 2041 Value: 0
Virtual address: 45023 Physical address: 16095 Value: -9
Virtual address: 44828 Physical address: 15900 Value: 0
Virtual address: 37304 Physical address: 16312 Value: 0
Virtual address: 24523 Physical address: 16587 Value: -14
Virtual address: 30978 Physical address: 16642 Value: 30
Virtual address: 46943 Physical address: 16991 Value: -41
Virtual address: 10025 Physical address: 17193 Value: 0
Virtual address: 62233 Physical address: 17433 Value: 0
Virtual address: 27594 Physical address: 17866 Value: 26
Virtual address: 36455 Physical address: 15719 Value: -103
Virtual address: 31717 Physical address: 18149 Value: 0
Virtual address: 7713 Physical address: 18209 Value: 0
Virtual address: 4623 Physical address: 6927 Value: -125
Virtual address: 4147 Physical address: 3123 Value: 12
Virtual address: 9448 Physical address: 9192 Value: 0
Virtual address: 49721 Physical address: 10041 Value: 0
Virtual address: 1355 Physical address: 18507 Value: 82
Virtual address: 8712 Physical address: 776 Value: 0
Virtual address: 44676 Physical address: 9860 Value: 0
Virtual address: 64373 Physical address: 10869 Value: 0
Virtual address: 46055 Physical address: 13031 Value: -7
Virtual address: 4607 Physical address: 18943 Value: 127
Virtual address: 27793 Physical address: 19089 Value: 0
Virtual address: 18264 Physical address: 19288 Value: 0
Virtual address: 62988 Physical address: 19468 Value: 0
Virtual address: 24757 Physical address: 13493 Value: 0
Virtual address: 36540 Physical address: 15804 Value: 0
Virtual address: 63777 Physical address: 19745 Value: 0
Virtual address: 6702 Physical address: 20014 Value: 6
Virtual address: 37186 Physical address: 16194 Value: 36
Virtual address: 20615 Physical address: 20359 Value: 33
Virtual address: 63076 Physical address: 19556 Value: 0
Virtual address: 54789 Physical address: 20485 Value: 0
Virtual address: 15918 Physical address: 20782 Value: 15
Virtual address: 7391 Physical address: 21215 Value: 55
Virtual address: 12400 Physical address: 21360 Value: 0
Virtual address: 17070 Physical address: 21678 Value: 16
Virtual address: 17724 Physical address: 21820 Value: 0
Virtual address: 30526 Physical address: 22078 Value: 29
Virtual address: 61362 Physical address: 22450 Value: 59
Virtual address: 6465 Physical address: 22593 Value: 0
Virtual address: 19152 Physical address: 22992 Value: 0
Virtual address: 11513 Physical address: 23289 Value: 0
Virtual address: 9937 Physical address: 23505 Value: 0
Virtual address: 14261 Physical address: 23733 Value: 0
Virtual address: 10619 Physical address: 379 Value: 94
Virtual address: 61297 Physical address: 22385 Value: 0
Virtual address: 31119 Physical address: 16783 Value: 99
Virtual address: 1449 Physical address: 18601 Value: 0
Virtual address: 49252 Physical address: 23908 Value: 0
Virtual address: 47383 Physical address: 24087 Value: 69
Virtual address: 10629 Physical address: 389 Value: 0
Virtual address: 23193 Physical address: 24473 Value: 0
Virtual address: 55555 Physical address: 24579 Value: 64
Virtual address: 5560 Physical address: 3000 Value: 0
Virtual address: 27701 Physical address: 18997 Value: 0
Virtual address: 37564 Physical address: 25020 Value: 0
Virtual address: 8788 Physical address: 852 Value: 0
Virtual address: 53055 Physical address: 25151 Value: -49
Virtual address: 23745 Physical address: 25537 Value: 0
Virtual address: 44417 Physical address: 25729 Value: 0
Virtual address: 61461 Physical address: 25877 Value: 0
Virtual address: 3203 Physical address: 26243 Value: 32
Virtual address: 59594 Physical address: 14026 Value: 58
Virtual address: 62068 Physical address: 26484 Value: 0
Virtual address: 30800 Physical address: 26704 Value: 0
Virtual address: 53634 Physical address: 27010 Value: 52
Virtual address: 4207 Physical address: 3183 Value: 27
Virtual address: 50845 Physical address: 1693 Value: 0
Virtual address: 22302 Physical address: 27166 Value: 21
Virtual address: 293 Physical address: 27429 Value: 0
Virtual address: 53369 Physical address: 6777 Value: 0
Virtual address: 24449 Physical address: 16513 Value: 0
Virtual address: 40355 Physical address: 11683 Value: 104
Virtual address: 49537 Physical address: 1921 Value: 0
Virtual address: 19095 Physical address: 22935 Value: -91
Virtual address: 61236 Physical address: 22324 Value: 0
Virtual address: 19457 Physical address: 27649 Value: 0
Virtual address: 5602 Physical address: 3042 Value: 5
Virtual address: 63850 Physical address: 19818 Value: 62
Virtual address: 31394 Physical address: 28066 Value: 30
Virtual address: 46940 Physical address: 16988 Value: 0
Virtual address: 54093 Physical address: 28237 Value: 0
Virtual address: 10163 Physical address: 17331 Value: -20
Virtual address: 2132 Physical address: 28500 Value: 0
Virtual address: 54261 Physical address: 28405 Value: 0
Virtual address: 14317 Physical address: 23789 Value: 0
Virtual address: 26976 Physical address: 28768 Value: 0
Virtual address: 26031 Physical address: 14511 Value: 107
Virtual address: 44302 Physical address: 25614 Value: 43
Virtual address: 827 Physical address: 1083 Value: -50
Virtual address: 11637 Physical address: 11125 Value: 0
Virtual address: 25051 Physical address: 12507 Value: 118
Virtual address: 6309 Physical address: 29093 Value: 0
Virtual address: 30890 Physical address: 26794 Value: 30
Virtual address: 1764 Physical address: 29412 Value: 0
Virtual address: 19165 Physical address: 23005 Value: 0
Virtual address: 62298 Physical address: 17498 Value: 60
Virtual address: 65340 Physical address: 1340 Value: 0
Virtual address: 28983 Physical address: 29495 Value: 77
Virtual address: 17125 Physical address: 21733 Value: 0
Virtual address: 55114 Physical address: 29770 Value: 53
Virtual address: 5201 Physical address: 15441 Value: 0
Virtual address: 37633 Physical address: 29953 Value: 0
Virtual address: 21240 Physical address: 2552 Value: 0
Virtual address: 1469 Physical address: 18621 Value: 0
Virtual address: 64060 Physical address: 30268 Value: 0
Virtual address: 37524 Physical address: 24980 Value: 0
Virtual address: 10922 Physical address: 30634 Value: 10
Virtual address: 39244 Physical address: 30796 Value: 0
Virtual address: 64327 Physical address: 10823 Value: -47
Virtual address: 46599 Physical address: 30983 Value: -127
Virtual address: 62366 Physical address: 17566 Value: 60
Virtual address: 30856 Physical address: 26760 Value: 0
Virtual address: 27379 Physical address: 31475 Value: -68
Virtual address: 57114 Physical address: 31514 Value: 55
Virtual address: 60756 Physical address: 31828 Value: 0
Virtual address: 58597 Physical address: 32229 Value: 0
Virtual address: 11775 Physical address: 11263 Value: 127
Virtual address: 54087 Physical address: 28231 Value: -47
Virtual address: 23920 Physical address: 32368 Value: 0
Virtual address: 17490 Physical address: 32594 Value: 17
Virtual address: 8672 Physical address: 32992 Value: 0
Virtual address: 14850 Physical address: 33026 Value: 14
Virtual address: 48421 Physical address: 5157 Value: 0
Virtual address: 28777 Physical address: 33385 Value: 0
Virtual address: 2202 Physical address: 28570 Value: 2
Virtual address: 51773 Physical address: 33597 Value: 0
Virtual address: 9937 Physical address: 23505 Value: 0
Virtual address: 9462 Physical address: 9206 Value: 9
Virtual address: 33618 Physical address: 33874 Value: 32
Virtual address: 56581 Physical address: 34053 Value: 0
Virtual address: 52742 Physical address: 34310 Value: 51
Virtual address: 65221 Physical address: 34757 Value: 0
Virtual address: 14918 Physical address: 33094 Value: 14
Virtual address: 19001 Physical address: 22841 Value: 0
Virtual address: 62259 Physical address: 17459 Value: -52
Virtual address: 38283 Physical address: 34955 Value: 98
Virtual address: 39452 Physical address: 35100 Value: 0
Virtual address: 938 Physical address: 1194 Value: 0
Virtual address: 12416 Physical address: 21376 Value: 0
Virtual address: 47769 Physical address: 35481 Value: 0
Virtual address: 1358 Physical address: 18510 Value: 1
Virtual address: 47135 Physical address: 9247 Value: 7
Virtual address: 2726 Physical address: 35750 Value: 2
Virtual address: 54628 Physical address: 35940 Value: 0
Virtual address: 65331 Physical address: 1331 Value: -52
Virtual address: 1944 Physical address: 36248 Value: 0
Virtual address: 17572 Physical address: 32676 Value: 0
Virtual address: 18578 Physical address: 36498 Value: 18
Virtual address: 57948 Physical address: 36700 Value: 0
Virtual address: 41736 Physical address: 36872 Value: 0
Virtual address: 29141 Physical address: 29653 Value: 0
Virtual address: 55860 Physical address: 37172 Value: 0
Virtual address: 51119 Physical address: 37551 Value: -21
Virtual address: 15412 Physical address: 37684 Value: 0
Virtual address: 61733 Physical address: 37925 Value: 0
Virtual address: 52952 Physical address: 34520 Value: 0
Virtual address: 60013 Physical address: 38253 Value: 0
Virtual address: 52441 Physical address: 38617 Value: 0
Virtual address: 8730 Physical address: 794 Value: 8
Virtual address: 62644 Physical address: 38836 Value: 0
Virtual address: 36036 Physical address: 39108 Value: 0
Virtual address: 61231 Physical address: 22319 Value: -53
Virtual address: 54219 Physical address: 28363 Value: -14
Virtual address: 46876 Physical address: 16924 Value: 0
Virtual address: 26634 Physical address: 39178 Value: 26
Virtual address: 1010 Physical address: 1266 Value: 0
Virtual address: 56291 Physical address: 11491 Value: -8
Virtual address: 27858 Physical address: 19154 Value: 27
Virtual address: 7079 Physical address: 167 Value: -23
Virtual address: 16577 Physical address: 39617 Value: 0
Virtual address: 41965 Physical address: 37101 Value: 0
Virtual address: 25390 Physical address: 39726 Value: 24
Virtual address: 649 Physical address: 7817 Value: 0
Virtual address: 23687 Physical address: 25479 Value: 33
Virtual address: 26327 Physical address: 8407 Value: -75
Virtual address: 37770 Physical address: 30090 Value: 36
Virtual address: 48850 Physical address: 40146 Value: 47
Virtual address: 55269 Physical address: 29925 Value: 0
Virtual address: 20960 Physical address: 40416 Value: 0
Virtual address: 63889 Physical address: 19857 Value: 0
Virtual address: 1722 Physical address: 29370 Value: 1
Virtual address: 49700 Physical address: 10020 Value: 0
Virtual address: 46511 Physical address: 40623 Value: 107
Virtual address: 25058 Physical address: 12514 Value: 24
Virtual address: 54231 Physical address: 28375 Value: -11
Virtual address: 53399 Physical address: 6807 Value: 37
Virtual address: 20496 Physical address: 20240 Value: 0
Virtual address: 45816 Physical address: 5880 Value: 0
Virtual address: 39825 Physical address: 15249 Value: 0
Virtual address: 54606 Physical address: 35918 Value: 53
Virtual address: 40731 Physical address: 40731 Value: -58
Virtual address: 12662 Physical address: 41078 Value: 12
Virtual address: 53026 Physical address: 25122 Value: 51
Virtual address: 63915 Physical address: 19883 Value: 106
Virtual address: 10997 Physical address: 30709 Value: 0
Virtual address: 17026 Physical address: 21634 Value: 16
Virtual address: 40648 Physical address: 41416 Value: 0
Virtual address: 12876 Physical address: 41548 Value: 0
Virtual address: 9512 Physical address: 41768 Value: 0
Virtual address: 36323 Physical address: 42211 Value: 120
Virtual address: 48874 Physical address: 40170 Value: 47
Virtual address: 63581 Physical address: 42333 Value: 0
Virtual address: 41748 Physical address: 36884 Value: 0
Virtual address: 42116 Physical address: 42628 Value: 0
Virtual address: 43507 Physical address: 42995 Value: 124
Virtual address: 12003 Physical address: 739 Value: -72
Virtual address: 8888 Physical address: 952 Value: 0
Virtual address: 16545 Physical address: 39585 Value: 0
Virtual address: 30047 Physical address: 43103 Value: 87
Virtual address: 8536 Physical address: 32856 Value: 0
Virtual address: 17013 Physical address: 21621 Value: 0
Virtual address: 40235 Physical address: 11563 Value: 74
Virtual address: 42330 Physical address: 43354 Value: 41
Virtual address: 48778 Physical address: 40074 Value: 47
Virtual address: 58849 Physical address: 43745 Value: 0
Virtual address: 40072 Physical address: 43912 Value: 0
Virtual address: 57650 Physical address: 44082 Value: 56
Virtual address: 26395 Physical address: 14619 Value: -58
Virtual address: 32920 Physical address: 13208 Value: 0
Virtual address: 12775 Physical address: 41191 Value: 121
Virtual address: 32550 Physical address: 44326 Value: 31
Virtual address: 64332 Physical address: 10828 Value: 0
Virtual address: 57011 Physical address: 44723 Value: -84
Virtual address: 27674 Physical address: 18970 Value: 27
Virtual address: 61976 Physical address: 26392 Value: 0
Virtual address: 24581 Physical address: 13317 Value: 0
Virtual address: 16144 Physical address: 44816 Value: 0
Virtual address: 38757 Physical address: 45157 Value: 0
Virtual address: 36062 Physical address: 39134 Value: 35
Virtual address: 55791 Physical address: 24815 Value: 123
Virtual address: 7718 Physical address: 18214 Value: 7
Virtual address: 65402 Physical address: 1402 Value: 63
Virtual address: 32832 Physical address: 13120 Value: 0
Virtual address: 37241 Physical address: 16249 Value: 0
Virtual address: 22318 Physical address: 27182 Value: 21
Virtual address: 4281 Physical address: 3257 Value: 0
Virtual address: 51604 Physical address: 45460 Value: 0
Virtual address: 41222 Physical address: 45574 Value: 40
Virtual address: 29255 Physical address: 45895 Value: -111
Virtual address: 41321 Physical address: 45673 Value: 0
Virtual address: 39423 Physical address: 30975 Value: 127
Virtual address: 20362 Physical address: 46218 Value: 19
Virtual address: 2544 Physical address: 10480 Value: 0
Virtual address: 6528 Physical address: 22656 Value: 0
Virtual address: 44613 Physical address: 9797 Value: 0
Virtual address: 24497 Physical address: 16561 Value: 0
Virtual address: 833 Physical address: 1089 Value: 0
Virtual address: 48821 Physical address: 40117 Value: 0
Virtual address: 45667 Physical address: 5731 Value: -104
Virtual address: 41637 Physical address: 46501 Value: 0
Virtual address: 12164 Physical address: 46724 Value: 0
Virtual address: 4978 Physical address: 46962 Value: 4
Virtual address: 13532 Physical address: 47324 Value: 0
Virtual address: 42676 Physical address: 47540 Value: 0
Virtual address: 5090 Physical address: 47074 Value: 4
Virtual address: 27504 Physical address: 17776 Value: 0
Virtual address: 62339 Physical address: 17539 Value: -32
Virtual address: 16686 Physical address: 47662 Value: 16
Virtual address: 30105 Physical address: 43161 Value: 0
Virtual address: 35957 Physical address: 39029 Value: 0
Virtual address: 57929 Physical address: 36681 Value: 0
Virtual address: 25324 Physical address: 48108 Value: 0
Virtual address: 8379 Physical address: 48315 Value: 46
Virtual address: 10870 Physical address: 30582 Value: 10
Virtual address: 41775 Physical address: 36911 Value: -53
Virtual address: 43375 Physical address: 42863 Value: 91
Virtual address: 3936 Physical address: 48480 Value: 0
Virtual address: 22083 Physical address: 48707 Value: -112
Virtual address: 31042 Physical address: 16706 Value: 30
Virtual address: 55813 Physical address: 37125 Value: 0
Virtual address: 38568 Physical address: 11944 Value: 0
Virtual address: 1545 Physical address: 29193 Value: 0
Virtual address: 3319 Physical address: 26359 Value: 61
Virtual address: 12109 Physical address: 46669 Value: 0
Virtual address: 40105 Physical address: 43945 Value: 0
Virtual address: 8305 Physical address: 48241 Value: 0
Virtual address: 15698 Physical address: 48978 Value: 15
Virtual address: 17842 Physical address: 21938 Value: 17
Virtual address: 3567 Physical address: 49391 Value: 123
Virtual address: 21480 Physical address: 49640 Value: 0
Virtual address: 21221 Physical address: 2533 Value: 0
Virtual address: 35554 Physical address: 49890 Value: 34
Virtual address: 54979 Physical address: 20675 Value: -80
Virtual address: 41431 Physical address: 45783 Value: 117
Virtual address: 3210 Physical address: 26250 Value: 3
Virtual address: 39122 Physical address: 50130 Value: 38
Virtual address: 22362 Physical address: 27226 Value: 21
Virtual address: 62037 Physical address: 26453 Value: 0
Virtual address: 16715 Physical address: 47691 Value: 82
Virtual address: 52669 Physical address: 50365 Value: 0
Virtual address: 63744 Physical address: 19712 Value: 0
Virtual address: 21962 Physical address: 50634 Value: 21
Virtual address: 43738 Physical address: 50906 Value: 42
Virtual address: 41534 Physical address: 46398 Value: 40
Virtual address: 25703 Physical address: 51047 Value: 25
Virtual address: 14716 Physical address: 51324 Value: 0
Virtual address: 41044 Physical address: 51540 Value: 0
Virtual address: 38642 Physical address: 12018 Value: 37
Virtual address: 40037 Physical address: 43877 Value: 0
Virtual address: 14664 Physical address: 51272 Value: 0
Virtual address: 29874 Physical address: 51890 Value: 29
Virtual address: 32966 Physical address: 13254 Value: 32
Virtual address: 43849 Physical address: 12617 Value: 0
Virtual address: 5997 Physical address: 14189 Value: 0
Virtual address: 1660 Physical address: 29308 Value: 0
Virtual address: 53649 Physical address: 27025 Value: 0
Virtual address: 20978 Physical address: 40434 Value: 20
Virtual address: 52720 Physical address: 50416 Value: 0
Virtual address: 51913 Physical address: 33737 Value: 0
Virtual address: 21267 Physical address: 49427 Value: -60
Virtual address: 40187 Physical address: 44027 Value: 62
Virtual address: 48954 Physical address: 52026 Value: 47
Virtual address: 35481 Physical address: 49817 Value: 0
Virtual address: 30673 Physical address: 22225 Value: 0
Virtual address: 1035 Physical address: 52235 Value: 2
Virtual address: 25866 Physical address: 14346 Value: 25
Virtual address: 32929 Physical address: 13217 Value: 0
Virtual address: 20616 Physical address: 20360 Value: 0
Virtual address: 50548 Physical address: 3700 Value: 0
Virtual address: 20481 Physical address: 20225 Value: 0
Virtual address: 62932 Physical address: 52692 Value: 0
Virtual address: 18451 Physical address: 36371 Value: 4
Virtual address: 63785 Physical address: 19753 Value: 0
Virtual address: 36663 Physical address: 52791 Value: -51
Virtual address: 31970 Physical address: 53218 Value: 31
Virtual address: 21530 Physical address: 53274 Value: 21
Virtual address: 20887 Physical address: 40343 Value: 101
Virtual address: 54916 Physical address: 20612 Value: 0
Virtual address: 37491 Physical address: 24947 Value: -100
Virtual address: 26145 Physical address: 8225 Value: 0
Virtual address: 11008 Physical address: 53504 Value: 0
Virtual address: 6844 Physical address: 20156 Value: 0
Virtual address: 30250 Physical address: 53802 Value: 29
Virtual address: 20151 Physical address: 4791 Value: -83
Virtual address: 11199 Physical address: 53695 Value: -17
Virtual address: 36746 Physical address: 52874 Value: 35
Virtual address: 37623 Physical address: 25079 Value: -67
Virtual address: 60766 Physical address: 31838 Value: 59
Virtual address: 48445 Physical address: 5181 Value: 0
Virtual address: 19145 Physical address: 22985 Value: 0
Virtual address: 2965 Physical address: 8853 Value: 0
Virtual address: 33220 Physical address: 54212 Value: 0
Virtual address: 4189 Physical address: 3165 Value: 0
Virtual address: 31091 Physical address: 16755 Value: 92
Virtual address: 25406 Physical address: 39742 Value: 24
Virtual address: 3979 Physical address: 48523 Value: -30
Virtual address: 41405 Physical address: 45757 Value: 0
Virtual address: 11331 Physical address: 23107 Value: 16
Virtual address: 50046 Physical address: 54398 Value: 48
Virtual address: 21407 Physical address: 49567 Value: -25
Virtual address: 51369 Physical address: 54697 Value: 0
Virtual address: 18681 Physical address: 36601 Value: 0
Virtual address: 64074 Physical address: 30282 Value: 62
Virtual address: 63766 Physical address: 19734 Value: 62
Virtual address: 22692 Physical address: 54948 Value: 0
Virtual address: 29343 Physical address: 45983 Value: -89
Virtual address: 53581 Physical address: 26957 Value: 0
Virtual address: 18051 Physical address: 55171 Value: -96
Virtual address: 49651 Physical address: 2035 Value: 124
Virtual address: 49492 Physical address: 1876 Value: 0
Virtual address: 12900 Physical address: 41572 Value: 0
Virtual address: 30143 Physical address: 43199 Value: 111
Virtual address: 24515 Physical address: 16579 Value: -16
Virtual address: 13743 Physical address: 55471 Value: 107
Virtual address: 37710 Physical address: 30030 Value: 36
Virtual address: 2505 Physical address: 10441 Value: 0
Virtual address: 13139 Physical address: 4179 Value: -44
Virtual address: 30534 Physical address: 22086 Value: 29
Virtual address: 17501 Physical address: 32605 Value: 0
Virtual address: 6435 Physical address: 22563 Value: 72
Virtual address: 3490 Physical address: 49314 Value: 3
Virtual address: 30492 Physical address: 22044 Value: 0
Virtual address: 9528 Physical address: 41784 Value: 0
Virtual address: 59657 Physical address: 7433 Value: 0
Virtual address: 20163 Physical address: 4803 Value: -80
Virtual address: 27456 Physical address: 17728 Value: 0
Virtual address: 23329 Physical address: 13601 Value: 0
Virtual address: 46241 Physical address: 55713 Value: 0
Virtual address: 52075 Physical address: 55915 Value: -38
Virtual address: 65269 Physical address: 34805 Value: 0
Virtual address: 38035 Physical address: 56211 Value: 36
Virtual address: 17452 Physical address: 32556 Value: 0
Virtual address: 41142 Physical address: 51638 Value: 40
Virtual address: 12719 Physical address: 41135 Value: 107
Virtual address: 6329 Physical address: 29113 Value: 0
Virtual address: 17336 Physical address: 56504 Value: 0
Virtual address: 61261 Physical address: 22349 Value: 0
Virtual address: 26162 Physical address: 8242 Value: 25
Virtual address: 46339 Physical address: 40451 Value: 64
Virtual address: 52074 Physical address: 55914 Value: 50
Virtual address: 63021 Physical address: 19501 Value: 0
Virtual address: 48633 Physical address: 5369 Value: 0
Virtual address: 57063 Physical address: 44775 Value: -71
Virtual address: 26270 Physical address: 8350 Value: 25
Virtual address: 26307 Physical address: 8387 Value: -80
Virtual address: 49500 Physical address: 1884 Value: 0
Virtual address: 63436 Physical address: 12236 Value: 0
Virtual address: 63619 Physical address: 42371 Value: 32
Virtual address: 18653 Physical address: 36573 Value: 0
Virtual address: 48786 Physical address: 40082 Value: 47
Virtual address: 27685 Physical address: 18981 Value: 0
Virtual address: 54925 Physical address: 20621 Value: 0
Virtual address: 24081 Physical address: 56593 Value: 0
Virtual address: 44804 Physical address: 15876 Value: 0
Virtual address: 39591 Physical address: 35239 Value: -87
Virtual address: 59359 Physical address: 57055 Value: -9
Virtual address: 41151 Physical address: 51647 Value: 47
Virtual address: 14000 Physical address: 57264 Value: 0
Virtual address: 20027 Physical address: 4667 Value: -114
Virtual address: 21998 Physical address: 50670 Value: 21
Virtual address: 16131 Physical address: 44803 Value: -64
Virtual address: 63023 Physical address: 19503 Value: -117
Virtual address: 18275 Physical address: 19299 Value: -40
Virtual address: 53500 Physical address: 6908 Value: 0
Virtual address: 54340 Physical address: 3908 Value: 0
Virtual address: 27093 Physical address: 28885 Value: 0
Virtual address: 7229 Physical address: 21053 Value: 0
Number of Translated Addresses = 500
Page Faults = 224
Page Fault Rate = 0.448
TLB Hits = 39
TLB Hit Rate = 0.078
